scripts/linters/pylint_extensions.py

```diff
@@ -1369,6 +1369,9 @@ class ImportOnlyModulesChecker(checkers.BaseChecker):
         ),
     }
 
+    # If import from any of these is made, it may not be a module.
+    EXCLUDED_IMPORT_MODULES = ['__future__', 'typing']
```

Comment:
Considering we have added __future__ here can we remove the flags from all the future imports? (I can create a separate PR if it's possible, can you create a github-issue for the same once this PR is approved?)

Reply:
I have made an [issue](https://github.com/oppia/oppia/issues/13326) for this.